Support/Testing/Travis/install.py

```diff
@@ -65,6 +65,7 @@
     android_platform = os.getenv('ANDROID_PLATFORM', "21")
     check_call('./Support/Scripts/prepare-android-toolchain.sh "%s:%s"' %
                (android_toolchains[target], android_platform), shell=True)
+    check_call('./Support/Scripts/prepare-android-ndk.sh', shell=True)
```

Comment:
This should be in the next commit instead, no?

Reply:
It could be, but I added it in this commit to make sure the script actually worked. That is to say, to make sure that the script didn't error out and die at some point. I don't think it's a big deal which commit it goes into.